src/main/java/com/foundation/trello/model/Card.java

```diff
@@ -2,8 +2,46 @@
 
 /**
  * This class works with JsonConvert class.
- * @author Andres Burgos
+ *
+ * @author Andres Burgos, Maday Alcala.
  */
 public class Card {
-    //Card class for jsonConverter.
+    private String id;
+    private String name;
+
+    /**
+     * This method is used to get id.
+     *
+     * @return a string.
+     */
+    public String getId() {
+        return id;
+    }
+
+    /**
+     * This method is used to set id.
+     *
+     * @param id the id parameter defines of input id.
+     */
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    /**
+     * This method is used to get name.
+     *
+     * @return a string.
```

Comment:
Please explain the content and use of String

Reply:
Solved.